eth_account/types.py

```diff
@@ -19,5 +20,6 @@
 PrivateKeyType = Union[Bytes32, int, HexStr, PrivateKey]
 
 AccessList = Sequence[Dict[str, Union[HexStr, Sequence[HexStr]]]]
+RLPStructuredAccessList = Sequence[Tuple[HexStr, Tuple[HexStr, ...]]]
```

Comment:
Let's use a `Sequence` here as well

```suggestion
RLPStructuredAccessList = Sequence[Tuple[HexStr, Sequence[HexStr, ...]]]
```

Reply:
updated